buildlib/tools/coverity.sh

```diff
@@ -135,4 +133,9 @@ else
 	cd "${ucx_build_dir}"
 fi
 
-run_coverity "$mode"
+run_coverity "$mode" || {
+	set +x
+	azure_log_error "Coverity found $nerrors issues: $cov_error_log"
```

Comment:
saving the errors in a shell variable could be problematic since their length is limited
maybe we can just execute `cov-format-errors --dir $cov_build --emacs-style` at a later stage?

Reply:
we can't execute it after unloading the modules so I created an error log file in case the are errors.